app/components/school_information/details/head_component.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl govuk-!-margin-bottom-6">
+      <%= @school.name %>
+    </h1>
+  </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-m"><%= @selected_cohort.start_year %> recruitment</h2>
+  </div>
+</div>
+<div class="govuk-grid-row govuk-!-margin-bottom-6">
+  <div class="govuk-grid-column-one-third">
+    <div class="dashboard-numbers key-number">
+      <p class="govuk-heading-xl">0</p>
+      <p class="govuk-heading-s">participants added by schools</p>
+    </div>
+  </div>
+  <div class="govuk-grid-column-one-third">
+    <div class="dashboard-numbers">
+      <p class="govuk-heading-xl">0</p>
+      <p class="govuk-heading-s">early career teachers</p>
+    </div>
+  </div>
+  <div class="govuk-grid-column-one-third">
+    <div class="dashboard-numbers">
+      <p class="govuk-heading-xl">0</p>
```

Comment:
Is there a ticket to make it dynamic? Can we link to it?

Reply:
Lead provider journey is still being spec out and changed by Al and Shani - so for now we don't have the details how this will change or if it will exist at all